service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/AllergyStructureMapper.java

```diff
@@ -181,14 +184,15 @@ private String buildReactionPertinentInformation(AllergyIntolerance allergyIntol
     }
 
     private String buildNotePertinentInformation(AllergyIntolerance allergyIntolerance) {
-        String notes = StringUtils.EMPTY;
-        if (allergyIntolerance.hasNote()) {
-            List<Annotation> annotations = allergyIntolerance.getNote();
-            notes = annotations.stream()
-                .map(Annotation::getText)
-                .collect(Collectors.joining(StringUtils.SPACE));
-        }
-        return notes;
+        return Stream.concat(
+            allergyIntolerance.hasNote() ? allergyIntolerance.getNote().stream() : Stream.empty(),
+            messageContext.getInputBundleHolder().getRelatedConditions(allergyIntolerance.getId())
+                .stream()
+                .map(Condition::getNote)
+                .flatMap(List::stream)
+        )
```

Comment:
Same as for immunization, swap the order of these arguments to prepend condition notes before allergy notes